change/@uifabric-fabric-website-2019-07-10-17-15-36-monaco-editor.json

```diff
@@ -0,0 +1,8 @@
+{
+  "comment": "Website changes to work with monaco editor state management",
+  "type": "minor",
+  "packageName": "@uifabric/fabric-website",
+  "email": "nishikaza@berkeley.edu",
```

Comment:
>nishikaza [](start = 12, length = 9)

use ms alias please

Reply:
I think this was just a change file that was auto-generated, I updated my primary email in Github for future commits

---
In reply to: [302628097](https://github.com/OfficeDev/office-ui-fabric-react/pull/9769#discussion_r302628097) [](ancestors = 302628097)

Reply:
Just so you know, even though these were auto-generated, once they are generated you can modify them under the `change` folder in your project, so you could potentially address these comments if you want 😄

Reply:
Updated :)

---
In reply to: [302672819](https://github.com/OfficeDev/office-ui-fabric-react/pull/9769#discussion_r302672819) [](ancestors = 302672819)